www/src/AdminBundle/Model/ImportTwine.php

```diff
@@ -0,0 +1,592 @@
+<?php
+/* Copyright (C) 2016 Michael Giesler
+ *
+ * This file is part of Dembelo.
+ *
+ * Dembelo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Dembelo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License 3 for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License 3
+ * along with Dembelo. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/**
+ * @package AdminBundle
+ */
+
+namespace AdminBundle\Model;
```

Comment:
Die Klasse würde ich unter dem Namespace "UseCase", "Import" oder "Command" ablegen. Für meinen persönlichen Geschmack befindet sich in dieser Klasse zu viel Logik, die nichts mit der Model-Schicht zu tun hat. 

Was hälst du denn von AdminBundle\Import\Twine ?